src/utils/route-handlers-middleware/handle-route-request.ts

```diff
@@ -0,0 +1,57 @@
+import { type NextRequest, NextResponse } from 'next/server';
+
+import { GRPCError, getHTTPStatusCode } from '../grpc/grpc-error';
+import logger from '../logger';
+
+import type {
+  CombineMiddlewareContextType,
+  GetAllMiddlewaresReturnTypes,
+  MiddlewareFunction,
+  RequestHandlerFunction,
+} from './route-handlers-middleware.types';
+
+async function handleRouteRequest<
```

Comment:
This function is being invoked as `routeHandlersMiddleware` in the API routes. You should change the names to be consistent throughout all the files.